Ball of Duty Server/Domain/Entities/CharacterSpecializations/Blaster.cs

```diff
@@ -6,9 +7,20 @@ public class Blaster : Character
     {
         private const int HP = 100;
         private const double SIZE = 50;
+        private const double HP_INCREASE_FACTOR = 0.1;
+        private const int HP_REGEN = 5;
 
-        public Blaster() : base(SIZE, HP, Specializations.BLASTER)
+        public Blaster() : base(SIZE, HP, Specializations.BLASTER, HP_REGEN)
         {
         }
+
+        public override void UpdateServerStats()
+        {
+            Health.Max = HP + (int)(Score * HP_INCREASE_FACTOR);
+            if (Health.Value > Health.Max)
```

Comment:
this is useless. you call UpdateServerStats through DecayScore before RegenHealth is called.
that means that despite this check Health.Value may still exceed Health.Max